src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo_container">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav_container">
+        <ul class="nav_container__list">
+          <li class="list_item">
+            <a
+              class="is-active list_item__link"
+              href="#">Apple
+            </a>
+          </li>
+          <li class="list_item">
+            <a class="list_item__link" href="#">Samsung
+            </a>
+          </li>
+          <li class="list_item">
+            <a class="list_item__link" href="#">Smartphones
+            </a>
```

Comment:
```suggestion
            <a class="list_item__link" href="#">
              Smartphones
            </a>
```
This formatting not fixed yet too